app/indexers/generic_indexer.rb

```diff
@@ -9,7 +9,9 @@ def generate_solr_document
     super.tap do |solr_doc|
       OregonDigital::GenericMetadata::PROPERTIES.map(&:to_s).each do |prop|
         attr = object.attributes[prop]
-        if attr.is_a? ActiveTriples::Relation
+        if %w[date_created date_uploaded].include? prop
+          solr_doc["#{prop}_tesim"] = [Date.today] if object.attributes[prop].nil?
```

Comment:
I don't think we should be setting values for these Hyrax-included properties, it should already be handled with things like https://github.com/samvera/hyrax/blob/master/lib/hyrax/transactions/steps/set_modified_date.rb

Reply:
Same here. We heavily customized everything, so we needed to rebuild the indexer. This is what drives the whole indexing now and it ignores the hyrax base stuff because it expects core metadata we removed.

Reply:
We stripped out Core metadata? Where? We just had a ticket to not do that: https://github.com/OregonDigital/OD2/issues/211

Even if we do, I don't know that the indexer should actually be setting the values. Is `Date.today` the exact same format that Hyrax is setting?